cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java

```diff
@@ -39,7 +48,15 @@
    * @param driverClass the JDBC driver class
    */
   public static void cleanup(Class<? extends Driver> driverClass) {
-    shutDownMySQLAbandonedConnectionCleanupThread(driverClass.getClassLoader());
+    ClassLoader pluginClassLoader = driverClass.getClassLoader();
+    if (pluginClassLoader == null) {
+      // This could only be null if the classLoader is the Bootstrap/Primordial classloader. This should never be the
+      // case since the driver class is always loaded from the plugin classloader.
+      LOG.warn("PluginClassLoader is null. This should never happen. Aborting cleanup.");
```

Comment:
We should remove "This should never happen.", since if that line is logged then it has happened.